src/transaction.ts

```diff
@@ -59,6 +59,7 @@ export interface RequestOptions {
   jsonOptions?: JSONOptions;
   gaxOptions?: CallOptions;
   maxResumeRetries?: number;
+  columnInfo?: object;
```

Comment:
Consider giving this a more specific type. For the other fields, it's easy to look up what fields are meant to be set in those options. Here, we have no idea.